ArduPlane/mode.cpp

```diff
@@ -86,3 +86,14 @@ bool Mode::enter()
     return enter_result;
 }
 
+bool Mode::is_vtol_man_throttle() const
+{
+    if (plane.quadplane.is_tailsitter() && 
```

Comment:
need a block comment here explaining what is going on

Reply:
added big comment